tests/micro/arduino/test_arduino_rpc_server.py

```diff
@@ -318,31 +354,48 @@ def test_byoc_microtvm(board, arduino_cli_cmd, microtvm_debug, workspace_dir):
         arduino_board=board,
         arduino_cli_cmd=arduino_cli_cmd,
         workspace_dir=workspace_dir,
+        serial_number=serial_number,
     )
 
 
 def _make_add_sess_with_shape(
-    model, arduino_board, arduino_cli_cmd, workspace_dir, shape, build_config
+    model,
+    arduino_board,
+    arduino_cli_cmd,
+    workspace_dir,
+    shape,
+    build_config,
+    serial_number: str = None,
 ):
     A = tvm.te.placeholder(shape, dtype="int8")
     C = tvm.te.compute(A.shape, lambda i: A[i] + A[i], name="C")
     sched = tvm.te.create_schedule(C.op)
     return _make_sess_from_op(
-        model, arduino_board, arduino_cli_cmd, workspace_dir, "add", sched, [A, C], build_config
+        model,
+        arduino_board,
+        arduino_cli_cmd,
+        workspace_dir,
+        "add",
+        sched,
+        [A, C],
+        build_config,
+        serial_number,
     )
 
 
 @pytest.mark.parametrize(
     "shape,",
     [
         pytest.param((1 * 1024,), id="(1*1024)"),
-        pytest.param((4 * 1024,), id="(4*1024)"),
-        pytest.param((16 * 1024,), id="(16*1024)"),
+        # pytest.param((4 * 1024,), id="(4*1024)"),
+        # pytest.param((16 * 1024,), id="(16*1024)"),
```

Comment:
Why did we comment these out?

Reply:
thanks for catching this. I will uncomment them